src/ScriptHandler.php

```diff
@@ -29,6 +29,15 @@ public static function populateEnv(Event $event)
         $envFileFactory = new EnvFactory();
 
         $processor = new Processor($event->getIO(), $envFileFactory);
+
+        if (!isset($config['example-file'])) {
```

Comment:
if there is no example file key - assume it is array of files

Reply:
I think it might be better to add a `files` key to the JSON so it is explicitly saying you can add multiple  `example-file`, `generated-file` pairs. Something like this:
```JSON
{
  "extra": {
    "env-populate": {
      "files": [
        {
          "example-file": "app/.env.dist",
          "generated-file": "app/.env"
        },
        {
          "example-file": "server/.env.dist",
          "generated-file": "server/.env"
        },
        {
          "example-file": "backup/.env.dist",
          "generated-file": "backup/.env"
        }
      ]
    }
  }
}
```

This code can then change to something like:
```php
if (isset($config['files'])) {
    foreach ($config['files'] as $file) {
         // Process the files one at a time
    }
} else {
    // Do the existing single file processing to maintain backwards compatibility
}
```